docs/persistence/mq.md

```diff
@@ -1,7 +1,7 @@
 # MQ
 
 MQ is supported on the nais platform on-premises and on GCP. 
-We recommend using kafka where possible and sensible. 
+
```

Comment:
Siden vi nå fjerner anbefalingen av Kafka her, hvordan kan bruke MQ uten å måtte snakke tilbake til on-prem?

Reply:
2 ting,
Synes det blir feil å ha kafka-anbefaling , når man skal lese mq doc'en, 
MQ kan kjøre lokalt i container for lokal utvikling på pc, og det kjører like bra i gcp  som på mainframe. Det er endel persistente asynkorne mønstre hvor mq kan være bedre valg enn kafka, men det får være opp til utviklerne, men jeg synes ikke setningen "use kafka...."  har noe å gjøre i mq teknisk doc. Kanskje i en annen doc hvor man kan si noe genrelt om begge persistente asynkrone mønstre som brukes i nav.